src/portal-components/DetailsPage.tsx

```diff
@@ -418,6 +494,13 @@ export default class DetailsPage extends React.Component<
       const injectedProps = injectPropsIntoConfig(value, el, {
         ...deepCloneOfProps,
       })
+
+      // For explorer 2.0, cannot assign key `lockedFacet` to deepCloneOfProps due to type errors,
+      // assign lockedFacet value directly to injectedProps only if resolveSynId.value is true
+      if (resolveSynId && resolveSynId.value) {
```

Comment:
I think you could use the null propagation operator to make this just:
resolveSynId?.value

Reply:
This is cool! And it is fixed. Thank you!